File: compiled_lang/cs/src/b00.variables.cs
using System;

class B00Variables
{
    static void Main()
    {
        int number = 5;
        char character = 'a';
        string text = "This is a string";

        string output = "Number is " + number + ".\n"
            + "Character is '" + character + "'.\n"
            + "String is \"" + text + "\".\n";

        Console.Write(output);
    }
}

File: lessons/compiled_lang/go/src/b00.variables.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	number := 5
	character := 'a'
	text := "This is a string"

	output := "Number is " + strconv.Itoa(number) + ".\n" +
		"Character is '" + string(character) + "'.\n" +
		"String is \"" + text + "\".\n"

	fmt.Print(output)
}
